### ISSUE TITLE
Jobs Console fails to launch

### ISSUE THREAD

**Issue by untoldone** [mhenrixon/sidekiq-unique-jobs#208]
```
root@df9ab509d516:/app# bundle exec jobs console
Use `keys '*', 1000 to display the first 1000 unique keys matching '*'
Use `del '*', 1000, true (default) to see how many keys would be deleted for the pattern '*'
Use `del '*', 1000, false to delete the first 1000 keys matching '*'
bundler: failed to load command: jobs (/usr/local/bundle/bin/jobs)
Errno::ENOENT: No such file or directory @ rb_sysopen - console
  /usr/local/lib/ruby/2.3.0/irb/magic-file.rb:8:in `initialize'
  /usr/local/lib/ruby/2.3.0/irb/magic-file.rb:8:in `open'
  /usr/local/lib/ruby/2.3.0/irb/magic-file.rb:8:in `open'
  /usr/local/lib/ruby/2.3.0/irb/input-method.rb:101:in `initialize'
  /usr/local/lib/ruby/2.3.0/irb/context.rb:85:in `new'
  /usr/local/lib/ruby/2.3.0/irb/context.rb:85:in `initialize'
  /usr/local/lib/ruby/2.3.0/irb.rb:426:in `new'
  /usr/local/lib/ruby/2.3.0/irb.rb:426:in `initialize'
  /usr/local/lib/ruby/2.3.0/irb.rb:381:in `new'
  /usr/local/lib/ruby/2.3.0/irb.rb:381:in `start'
  /usr/local/bundle/gems/sidekiq-unique-jobs-4.0.18/lib/sidekiq_unique_jobs/cli.rb:24:in `console'
  /usr/local/bundle/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
  /usr/local/bundle/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
  /usr/local/bundle/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
  /usr/local/bundle/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
  /usr/local/bundle/gems/sidekiq-unique-jobs-4.0.18/bin/jobs:6:in `<top (required)>'
  /usr/local/bundle/bin/jobs:17:in `load'
  /usr/local/bundle/bin/jobs:17:in `<top (required)>'
```

Let me know if any more info would help debug. Running within Docker with version sidekiq-unique-jobs (4.0.18)

**Comment by mhenrixon** [comment]
Really weird, when I try it out in the `rails_example` folder the following happens ![rails_example ruby 2017-02-13 09-26-05](https://cloud.githubusercontent.com/assets/106795/22875924/7eac009a-f1ce-11e6-9175-e1bbaa729abd.png)

**Comment by untoldone** [comment]
Here's a simple repo. Only prereq is Docker.

```
docker run -it rails

# Wait for new prompt. In new prompt that prior command started:

mkdir app
cd app
rails new .

echo "gem 'sidekiq'" >> Gemfile
echo "gem 'sidekiq-unique-jobs'" >> Gemfile
bundle install
bundle exec jobs console
```

Note this repro doesn't include a Redis instance but given the error, it doesn't seem we make it that far.

**Comment by mhenrixon** [comment]
Fixed in version 5.0.0
